drivers/include/bme680.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2019 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bme680 BME680 Temperature/Humidity/Pressure/Gas sensor
+ * @ingroup     drivers_sensors
+ * @brief       Driver for the Bosch BME680  sensor
+ *
+ * @{
+ * @file
+ * @brief       Interface definition for the Bosch BME680 sensor
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef BME680_H
+#define BME680_H
+
+#include "periph/i2c.h"
+#include "periph/spi.h"
+
+#include "bme680_internal.h"
```

Comment:
```suggestion
#include "bme680_hal.h"
#include "bme680_defs.h"
#include "bme680_internal.h"
```
Avoids that the application has to include it. Makes the usage of the driver easier,